routes/web.php

```diff
@@ -59,10 +58,12 @@
 Route::get('/vragenantwoorden/vraagformulier', [FAQController::class, 'questionform']);
 Route::post('/vragenantwoorden/submit', [FAQController::class, 'submit']);
 Route::get('/vragenantwoorden', [FAQController::class, 'viewquestions']);
+
 //Event routes
 Route::resource('events', EventController::class);
 Route::get('/evenement/{event}/details', [EventController::class, 'show'])->name('eventsDetails');
-
+Route::get('evenement/export', [EventController::class, 'exportIcal']);
+Route::get('evenement/sortByDate/{date}', [EventController::class, 'sortByDate'])->name('evenement.sortByDate');
```

Comment:
waar is deze route voor precies?

Reply:
De bovenste is voor het exporteren van een ical bestand, de onderste had ik toegevoegd maar was uiteindelijk niet meer nodig.